src/errorpage.cc

```diff
@@ -1401,21 +1397,19 @@ SBuf
 ErrorState::buildBody()
 {
     assert(page_id > ERR_NONE && page_id < error_page_count);
+    errLanguage.reset();
 
 #if USE_ERR_LOCALES
     /** error_directory option in squid.conf overrides translations.
      * Custom errors are always found either in error_directory or the templates directory.
      * Otherwise locate the Accept-Language header
      */
     if (!Config.errorDirectory && page_id < ERR_MAX) {
-        if (err_language && err_language != Config.errorDefaultLanguage)
-            safe_free(err_language);
```

Comment:
AFAICT, this code was trying to reset err_language in case it was set earlier. I moved the corresponding reset earlier and made it unconditional. Most likely, none of this matters because err_language is probably always nil when buildBody starts generating the error page. I suspect that the removed condition was always false and its replacement reset() code above does nothing except clarifying our intent to start from scratch as far as errLanguage computation is concerned.

This comment does not request any PR changes.